cafebabel/templates/home.html

```diff
@@ -21,5 +22,19 @@
   {{ meta('name=twitter:url', request.url) }}
 {% endblock %}
 
+{% block body_class %}home{% endblock %}
+{% block body %}
+{% if articles %}
+  <h2>Editor's pick</h2>
+  {{ card(articles|first) }}
+{% endif %}
 
-{% block body %}{% endblock %}
+{% if articles|length > 2 %}
+  <h2>Latest</h2>
+  {{ card(articles[1]) }}
+
+  {% for article in articles[3:] %}
```

Comment:
Why not directly iterating over `[2:]` and remove the line above?

Reply:
good question. @vinyll ?